### ISSUE TITLE
Initial setup - getting partial data

### ISSUE THREAD

**Issue by JJJ77TX** [nebulous/infinitude#152]
I'm pleased that I have this running and passing data to grafana now, but unfortunately I'm only getting a fraction of the data to come through.  I assume that now I get to spend some quality time reading about registers to try to figure out how to get the other items reporting in (like compressor speed or zone temperatures).  I'm running version 4.18 on my connex thermostat.

Any tips to help me get started would be appreciated.

And yes, I created my github account for the purpose of sending this message.  I'm definitely a noob.
[AC-data.xlsx](https://github.com/nebulous/infinitude/files/9137525/AC-data.xlsx)
.

**Comment by nebulous** [comment]
Hi, @JJJ77TX - I think you may be looking for the https://github.com/dulitz/finitude project.

**Comment by dulitz** [comment]
Yes, this looks like "finitude" which I abandoned for a while. Picking it up again now. Please file an issue over there.

**Comment by JJJ77TX** [comment]
nebulous, thanks for correcting me on that.  This is definitely embarrassing.  I used your hardware page as a guide to wiring into the ABCD bus (RS-485 plus 24VAC).  I ended up in the wrong project after double checking the voltage on the D terminal yesterday using this page:  https://github.com/nebulous/infinitude/wiki/Infinity-Protocol-Hardware.  Still the best page for that purpose in my opinion.
Thanks.
